webpack/config.js

```diff
@@ -1,9 +1,8 @@
-require("dotenv").config();
```

Comment:
I scratched my head over this for a while. Is .env used by webpack in ways that I don't understand? This line was causing webpack to fail to build in CI where .env does not exist. Why doesn't the Heroku Dockerfile build fail in the same way?